pepper-apis/src/main/java/org/broadinstitute/ddp/db/dto/ActivityInstanceDto.java

```diff
@@ -82,7 +82,7 @@ public Long getFirstCompletedAt() {
         return firstCompletedAt;
     }
 
-    public boolean isReadonly() {
+    public Boolean isReadonly() {
```

Comment:
Instead of propagating `Boolean` for various getters, could we keep the returned value as `boolean` and implement getter methods as `return isHidden != null ?  isHidden :  false`?  That might simplify how clients interpret things.

Reply:
I guess you meant `isReadonly`. Other than that, I agree that this can simplify things for clients (given that we are sure that we want to treat `null` as `false`). 

This, however, won't work for `ActivityInstanceDto` which is passed to `ActivityInstanceUtilTest.isReadonly(...)` which, in turn, bases its logic on the nullability of the `ActivityInstanceDto.isReadonly()` result.

But for other models it should be fine. Done.

Reply:
I think I will be confused if it always return true/false even thought it's nullable. Like @raiks2 said, sometimes you _do_ want to check for the null. How about we allow both? `isReadonly()` is the convenience method to simplify things for clients, while `getReadonly()` lets you get the raw value:
```
public boolean isReadonly {}
public Boolean getReadonly {}
```

Reply:
Cool!

Reply:
Done.